### DISCUSSION TITLE
图片可以渲染吗？

### DISCUSSION THREAD

**Discussion by gkongy** [F9y4ng/GreasyFork-Scripts#177] [问答专区(Question / Answer)]
比如较多使用的模糊阴影？

**Comment by Xia0xia0Xia0** [comment]
图片不是字体，应该是不可以，否则也不会叫字体渲染。

**Comment by F9y4ng** [comment]
不会渲染图片，也不会为图片添加其他装饰效果。毕竟图片与字体渲染无关。

**Reply by gkongy** [reply]
文字加图片都渲染就可以直接编辑复制到发布了

**Reply by F9y4ng** [reply]
> 文字加图片都渲染就可以直接编辑复制到发布了

如何定义哪些图片需要渲染哪些不需要渲染？在无数的网站中如何定义规则？这个需求有点不切合实际，也完全脱离字体渲染优化的范围。

所以，我建议你可以选择其他针对图片特效的脚本来尝试一下。
